src/useUpdateEffect/index_en.mdx

```diff
@@ -0,0 +1,43 @@
+---
+name: useUpdateEffect
+route: /useUpdateEffect
+menu: 'basic hooks'
+edit: false
+sidebar: true
+---
+import JackBox from 'jackbox';
+
+import Demo1 from './demo/demo1';
+import Demo1CodeTsx from '!raw-loader!./demo/demo1.tsx';
+import Demo1CodeJsx from '!raw-loader!./demo/demo1.jsx';
+
+# useUpdateEffect
+
+A hook only works after component mounted.
+
+**Core Characteristics**
+
+* Suitable for use in hooks business scenarios
+* Optional effect hook cleanup mechanism to remove side effects
+
+## Examples
+
+<JackBox jsCode={Demo1CodeJsx} tsCode={Demo1CodeTsx} demoName='basic usage' description='It is exactly the same in use as useEffect, except that the life cycle is placed after the component didMount'>
```

Comment:
This hook is exactly the same as useEffect, except it omits the first render and only runs when dependencies update.

Reply:
ok